src/coffee/directives/api/map.coffee

```diff
@@ -259,7 +259,10 @@ angular.module('uiGmapgoogle-maps.directives.api')
               scope.$watch toWatch, (newValue,oldValue) ->
                 watchItem = @exp
                 return  if _.isEqual(newValue,oldValue)
-                opts.options = newValue
```

Comment:
style was never updating before.. but honestly the style attribute should disappear as options is enough.